gui/MetadataView.cxx

```diff
@@ -281,7 +281,10 @@ void MetadataView::updateMetadata(kv::metadata_vector const& mdVec)
 {
   QTE_D();
 
-  for (auto const k : d->itemIds())
+  auto traits = kv::metadata_traits{};
```

Comment:
This doesn't seem to be "used"? It is rather odd that `tag_to_name` is not a static method... (Although that would require KWIVER changes...)

Reply:
Yes, I had the same thought.  It should be static, maybe I'll change that later.